### PR TITLE
update(Adhan-time): Add search view 

### PR DESCRIPTION
**Pull request by mzaien**
## Description

Update to adhan prayer time extension to add a view mode addressing this [issue](https://github.com/raycast/extensions/issues/3417) 
while doing the new view I refactored the code to be more cleaner 
## Screencast

CleanShot 2022-11-06 at 06 20 15

## Checklist

- [ x ] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [ x ] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [ x ] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [ x ] I checked that files in the `assets` folder are used by the extension itself
- [ x ] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by pernielsentikaer** [comment]
Hi @mzaien 👋 

It seems like you have some conflicts, could you resolve them 😊

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

I left a single comment for you to address 😊

File: extensions/adhan-time/package.json [hunk 1]
```diff
@@ -12,8 +12,15 @@
...
-      "name": "adhan",
-      "title": "View Prayer Time",
+      "name": "adhan-search",
+      "title": "Search Prayer Time",
+      "description": "View adhan prayer times directly from Raycast",
+      "mode": "view",
+      "interval": "24h"
+    },
+    {
+      "name": "adhan-menu",
```
[the remark this answers is not in the document]
Comment (mzaien) [reply extensions/adhan-time/package.json:22]: Nice catch 👍, thank you will change it soon

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Comment by mzaien** [comment]
> https://raycast.com/mzaien/adhan-time

Thanks

**Pull request merged**